MULTIPLATFORM.md

```diff
@@ -0,0 +1,325 @@
+
+    include ':greeting'
+    include ':greeting:common'
+    include ':greeting:android'
+    include ':greeting:ios'
+
+Here we declare all subprojects for our `greeting` multiplatform library. All other multiplatform libraries included
+in the project also must be declared here.
+
+Note that both iOS and Android applications are not included in the root Gradle build. They are represented by
+independent builds which are managed by specific IDEs (Android Studio and XCode). Such an approach makes work with
+these builds easier from these IDEs.
+
+As for IDE for other parts of the project, IntelliJ IDEA is recommended to be used.
+
+> Note: Kotlin/Native is not supported by IntelliJ IDEA so the only IDE to develop Kotlin/Native subprojects is
+CLion. But at the moment CLion has no Gradle integration. As a workaround you can create a CLion Cmake project
+from a Kotlin/Native Gradle one. Just run the `generateCMake` task for this project.
+
+As the final step create empty `build.gradle` files in the root directory of the project in all subprojects
+included in `settings.gradle`. After all these actions the project structure will be the following (files
+generated by the Gradle wrapper are not shown):
+
+    application/
+    ├── androidApp/
+    ├── iosApp/
+    ├── greeting/
+    │   ├── android/
+    │   │   └── build.gradle
+    │   ├── common/
+    │   │   └── build.gradle
+    │   ├── ios/
+    │   |   └── build.gradle
+    |   └── build.gradle
+    ├── build.gradle
+    └── settings.gradle
+
+Now we have a basic structure of the project and can proceed to implementing of the multiplatform library.
+
+### 2. Multiplatform library
+
+We need to add buildscript dependencies to be able to use Kotlin plugins for Gradle in our build. Open
+`build.gradle` in the `greeting` directory and put into it the following snippet:
+
+    // Set up a buildscript dependency on the Kotlin plugin.
+    buildscript {
+        // Specify a Kotlin version you need.
+        ext.kotlin_version = '1.2.31'
+
+        repositories {
+            google()
+            jcenter()
+            maven { url "https://dl.bintray.com/jetbrains/kotlin-native-dependencies" }
+        }
+
+        // Specify all the plugins used as dependencies
+        dependencies {
+            classpath 'com.android.tools.build:gradle:3.1.0'
+            classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+            classpath "org.jetbrains.kotlin:kotlin-native-gradle-plugin:0.6.2"
+
+        }
+    }
+
+    // Set up compilation dependency repositories for all projects.
+    subprojects {
+        repositories {
+            google()
+            jcenter()
+        }
+    }
+
+Now all subprojects of the library can use Kotlin plugins.
+
+#### 2.1 Common subproject
+
+The `common` subproject contains a platform-independent code. Let's create `common/src/main/kotlin/common.kt` and
+add some functionality into it:
+
+    // greeting/common/src/main/kotlin/common.kt
```

Comment:
I would simply remove that commend.
It does not bring any value. 
The sentences above  says all (where to put the file into)